lib-object-oriented/line.js

```diff
@@ -1,34 +1,77 @@
 import Tag from './tag'
 
 export default class Line {
-  constructor( content ) {
+  constructor({ content, type }) {
     this.content = content
+    this.tagType = type
+    this.symbolCount = 0
+    this.getSymbolCount()
+    this.newLinesToSpaces()
+  }
+
+  newLinesToSpaces() {
+    const tagType = this.tagType,
+          content = this.content,
+          symbolCount = this.symbolCount
+
+    const findNewLine = element => {
+      return element.type === 'new_line'
+    }
+
+    if ( tagType !== '`' || ( tagType === '`' && symbolCount === 1 ) ) {
```

Comment:
This appears to be another opportunity to re-use the token constants defined elsewhere.  I would also consider making this another method - there's __a lot__ going on in this function, and decomposing the problem into some appropriately named functions could help readers parse this a little easier.  I also already forgot that `tagType` was a re-declaration of `this.tagType`, so had to scroll up to confirm where this came from to begin to understand this code.
```js
if( isValidTag() ) // I made that up I actually have no idea what this check is for
```